docs/tables/aws_guardduty_finding/queries.md

```diff
@@ -0,0 +1,384 @@
+where
+  (service ->> 'feature_name') = 'MalwareProtection'
+  and (resource ->> 'resource_type') = 'Instance'
+order by
+  tp_timestamp desc;
+```
+
+```yaml
+folder: GuardDuty
+```
+
+## Operational Examples
+
+### Runtime Monitoring Findings
+
+Investigate security events related to runtime behavior in containerized (EKS) and virtual machine (EC2) environments. This analysis helps detect suspicious processes, unauthorized activities, and potential security violations during program execution.
+
+```sql
+select
+  tp_timestamp,
+  tp_index as account_id,
+  region,
+  title,
+  type,
+  severity,
+  description,
+  (resource ->> 'resource_type') as resource_type
+from
+  aws_guardduty_finding
+where
+  (service ->> 'feature_name') = 'RuntimeMonitoring'
+order by
+  tp_timestamp desc;
+```
+
+```yaml
+folder: GuardDuty
+```
+
+### IAM Credential Misuse
+
+Track potential compromises or misuse of IAM credentials by analyzing AccessKey-related findings. This query helps identify unauthorized access attempts and potential credential leaks across your AWS environment.
+
+```sql
+select
+  tp_timestamp,
+  tp_index as account_id,
+  region,
+  title,
+  type,
+  severity,
+  (resource ->> 'access_key_details') as access_key_details
+from
+  aws_guardduty_finding
+where
+  (resource ->> 'resource_type') = 'AccessKey'
+order by
+  severity desc,
+  tp_timestamp desc;
+```
+
+```yaml
+folder: GuardDuty
+```
+
+## Volume Examples
+
+### High Volume of Findings
+
+Identify AWS accounts experiencing unusually high numbers of security findings (>50 per day), which may indicate targeted attacks, misconfiguration, or systematic security issues requiring immediate attention.
+
+```sql
+select
+  account_id,
+  count(*) as finding_count,
+  date_trunc('day', created_at) as finding_day
+from
+  aws_guardduty_finding
+group by
+  account_id,
+  finding_day
+having
+  count(*) > 50
+order by
+  finding_count desc;
+```
+
+```yaml
+folder: GuardDuty
+```
+
+### Repeat Findings
+
+Track recurring security issues by identifying findings that appear multiple times (>5 occurrences), helping teams focus on persistent security problems that require systematic resolution rather than one-off fixes.
+
+```sql
+select
+  type,
+  title,
+  count(*) as occurrences
+from
+  aws_guardduty_finding
+group by
+  type,
+  title
+having
+  count(*) > 5
+order by
+  occurrences desc;
+```
+
+```yaml
+folder: GuardDuty
+```
+
+## Baseline Examples
+
+### New Finding Types
+
+Monitor the emergence of new security finding types within the last 7 days, helping security teams quickly identify and respond to novel threats or attack patterns in your AWS environment.
+
+```sql
+select
+  type,
+  min(created_at) as first_seen,
+  count(*) as occurrence_count
+from
+  aws_guardduty_finding
+group by
+  type
+having
+  min(created_at) > current_date - interval '7 days'
+order by
+  first_seen desc;
+```
+
+```yaml
+folder: GuardDuty
+```
+
+### Abnormal Resource Behavior
```

Comment:
Should this go under the Detections section instead, since it doesn't use time windowing or comparison to earlier results?